azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/lobby/ConnectingLobbyOverlayView.kt

```diff
@@ -39,6 +41,12 @@ internal class ConnectingLobbyOverlayView : LinearLayout {
     ) {
         this.viewModel = viewModel
 
+        if (viewModel.getCallType() == CallType.ONE_TO_N_CALL_OUTGOING) {
```

Comment:
If can be embedded in getString() for slight reduction of duplication.

Reply:
As per discussion we will fix for GA.